src/AppBundle/Controller/UserGroupCollectionController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\UserGroupCollection;
+use AppBundle\Form\Type\UserGroupCollectionType;
+use AppBundle\Service\UserGroupCollectionManager;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class UserGroupCollectionController extends BaseController
+{
+    public function createUserGroupCollectionAction(Request $request, UserGroupCollection $userGroupCollection = null)
+    {
+        if ($isCreate = $userGroupCollection === null) {
+            $userGroupCollection = new UserGroupCollection();
+        }
+        $isEditable = !$userGroupCollection->isDeletable();
+
+        $em = $this->getDoctrine()->getManager();
+        $bolkNames = $em
+            ->getRepository('AppBundle:AssistantHistory')
+            ->findAllBolkNames();
+
+
+        $form = $this->createForm(UserGroupCollectionType::class, $userGroupCollection, array(
+            'bolkNames' => $bolkNames,
+            'isEdit' => $isEditable,
+        ));
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            try {
+                $this->get(UserGroupCollectionManager::class)->initializeUserGroupCollection($userGroupCollection);
+            } catch (\InvalidArgumentException $e) {
+                $this->addFlash("danger", $e->getMessage());
+                return $this->redirect($this->generateUrl('usergroup_collection_create'));
+            } catch (\UnexpectedValueException $e) {
+                $this->addFlash("danger", $e->getMessage());
+                return $this->redirect($this->generateUrl('usergroup_collection_create'));
```

Comment:
Hvis brukeren blir redirected vil alle data brukeren har fylt inn i formen bli slettet. For å beholde alle utfylte data må dette returneres:
```php
return $this->render('usergroup_collection/usergroup_collection_create.html.twig', array(
            'form' => $form->createView(),
            'isCreate' => $isCreate,
            'userGroupCollection' => $userGroupCollection,
));
```